docs/platforms/android/releases/index.mdx

```diff
@@ -0,0 +1,41 @@
+---
+title: Release Channels
+description: "Learn about the SDK's release channels and choose the right one."
```

Comment:
Since this change is only applicable to (specific, i.e. the "big") Mobile SDKs I didn't want to make it sound like this will be used across all of Sentry